MAUI/DataGrid/column-resizing.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: Column Resizing in .NET MAUI DataGrid control | Syncfusion
+description: Learn all about column reizing support in Syncfusion MAUI DataGrid (SfDataGrid) control and more here.
+platform: MAUI
+control: SfDataGrid
+documentation: ug
+---
+
+# Resizing columns in MAUI DataGrid (SfDataGrid)
+
+The `SfDataGrid` allows you to resize the columns by tapping and dragging the right border of the column headers. Resizing can be enabled or disabled by setting the `SfDataGrid.AllowResizingColumns` property. A resizing indicator is displayed while resizing a column.
+
+N> Resizing considers `DataGridColumn.MinimumWidth` and `DataGridColumn.MaximumWidth` of the column and will not exceed the minimum and maximum width constraints.
+
+{% tabs %}
+{% highlight xaml %}
+
+    <syncfusion:SfDataGrid  x:Name="dataGrid"
+                        AllowResizingColumns="True"
+                        ItemsSource="{Binding Orders}" />
+
+{% endhighlight %}
+{% highlight c# %}
+
+        dataGrid.AllowResizingColumns = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+![DataGrid with column resizing](Images\column-resizing\maui-datagrid-column-resizing.gif)
+
+The column width can be changed by tapping and dragging the resizing indicator.
+
+N> The resizing indicator appears when tapping the right corner of the column header.
```

Comment:
Remove the line since we already explained this in the description itself.

Reply:
Addressed the review concerns provided.